core/src/test/scala/org/broadinstitute/dsde/rawls/entities/EntityShardingSpec.scala

```diff
@@ -72,7 +72,7 @@ class EntityShardingSpec extends AnyFlatSpec with Matchers
   // checks row counts for each shard, plus the _archived table
   def checkShardCounts(expected: Map[ShardId, Int] = Map()): Unit = {
     // default to 0 for all shards plus the archive table
-    val default: Map[ShardId, Int] = (allShards + "archived").map(_ -> 0).toMap
+    val default: Map[ShardId, Int] = (allShards).map(_ -> 0).toMap
```

Comment:
and this prevents this specific test class from trying to test the `ENTITY_ATTRIBUTE_archived` table